packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -56,16 +62,20 @@
             name="address-postcode"
             :label-text="$t('labels.postcode')"
             :has-error="!isAddressPostcodeValid"
+            aria-describedby="postcode-error"
+            @blur="isFieldValid('postcode')"
             @input="updateAddressDetails({ ['postcode']: $event })">
             <template #error>
                 <error-message
                     v-if="isAddressPostcodeEmpty"
+                    id="postcode-error"
                     data-js-error-message
                     data-test-id="error-address-postcode-empty">
                     {{ $t('validationMessages.postcode.requiredError') }}
                 </error-message>
                 <error-message
                     v-else-if="!isAddressPostcodeValid"
+                    id="postcode-error"
```

Comment:
Question rather than suggestion: would it work if you moved the shared attributes like `id` and `data-js-error-message` to the `template` tag?

Reply:
just tried and it doesn't work sadly - looks like they both need to be after the two conditions